repositories/migrations/20240124111516_create_analytics_schema.sql

```diff
@@ -0,0 +1,35 @@
+-- +goose Up
+-- +goose StatementBegin
+-- create and make default the marble schema
+CREATE SCHEMA IF NOT EXISTS analytics;
+
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA analytics TO postgres;
+
+DO $ $ BEGIN CREATE USER analytics WITH PASSWORD 'default';
```

Comment:
This routine will create the user analytics if it doesn't already exists. The expectation is that, in our cloud environment at least, we will have created the analytics user in cloudsql, and stored the password in secret manager/1password. But this should still work in local dev mode or for open source clients.